frontend/service/agents/Memeooor.ts

```diff
@@ -178,11 +182,11 @@ export abstract class MemeooorBaseService extends StakedAgentService {
   static getStakingContractDetails = async (
     stakingProgramId: StakingProgramId,
     chainId: EvmChainId,
-  ): Promise<StakingContractDetails> => {
+  ): Promise<StakingContractDetails | undefined> => {
     const { multicallProvider } = PROVIDERS[chainId];
 
-    const { contract: stakingTokenProxy } =
-      STAKING_PROGRAMS[chainId][stakingProgramId];
+    const stakingTokenProxy = BASE_STAKING_PROGRAMS[stakingProgramId]?.contract;
```

Comment:
hmm if we know this is base chain, why to pass chainId then? can't we just do smth like:
```
if (!Object.keys(STAKING_PROGRAMS[chainId]).includes(stakingProgramId)) return
```